backend/src/packages/shifts/shift.service.ts

```diff
@@ -37,6 +38,10 @@ class ShiftService implements IService {
     return shifts.map((it) => ShiftEntity.initialize(it).toObject());
   }
 
+  public getAllStartedWithTrucks(): Promise<TruckEntity[]> {
```

Comment:
am I right that now on home page we display only those trucks which have current shifts?

Reply:
Yeah, that's right. But this can also include trucks that are on shift but have already taken an order. According to our specification there is a label on the track card that shows the status of the track, so I don't filter the busy tracks. We need to discuss this issue and decide how to handle it.